cpp/include/cudf/strings/capitalize.hpp

```diff
@@ -30,21 +30,33 @@ namespace strings {
 /**
  * @brief Returns a column of capitalized strings.
  *
- * Any null string entries return corresponding null output column entries.
+ * If the `delimiters` is an empty string, then only the first character of each
+ * row is capitalized. Otherwise, a non-delimiter character is capitalized after
+ * any delimiter character is found.
  *
  * @code{.pseudo}
  * Example:
- * input = ["tesT1", "a Test", "Another Test"];
+ * input = ["tesT1", "a Test", "Another Test", "a\tb"];
  * output = capitalize(input)
- * output is ["Test1", "A test", "Another test"]
+ * output is ["Test1", "A test", "Another test", "A\tb"]
+ * output = capitalize(input, " ")
+ * output is ["Test1", "A Test", "Another Test", "A\tb"]
+ * output = capitalize(input, " \t")
+ * output is ["Test1", "A Test", "Another Test", "A\tB"]
  * @endcode
  *
- * @param[in] input String column.
- * @param[in] mr Device memory resource used to allocate the returned column's device memory
+ * Any null string entries return corresponding null output column entries.
+ *
+ * @throw cudf::logic_error if `delimiter.is_valid()` is  `false`.
```

Comment:
Is this a correct behavior? Should invalid delimiter and empty delimiter produce the same output?

Reply:
An invalid delimiter should throw an error as described here. My understanding is the Spark code will always pass a single space ' '  string here. An empty delimiter is for the original behavior as expected from cudf and consistent with Pandas.